### ISSUE TITLE
It doesn't preserve file.stat

### ISSUE THREAD

**Issue by fabiospampinato** [backflip/gulp-resolve-dependencies#9]
I faced a problem with `gulp-newer` where it cannot compare the files since that .stat property is missing after running this plugin.

**Comment by backflip** [comment]
Looks like `vinyl` has a `stats` option: https://github.com/wearefractal/vinyl#optionsstat. I will check whether I can use this.

**Comment by backflip** [comment]
Could you check whether the latest commit (not yet published to npm) solved your issue?

```
npm i gulp-resolve-dependencies@https://github.com/backflip/gulp-resolve-dependencies.git#6a2cb53c15ceb4a8bd487f49d54665344bca4928
```

**Comment by fabiospampinato** [comment]
It does, thanks!

**Comment by backflip** [comment]
Okay, I will release this as soon as possible. Don't really know yet whether this is going to be 2.2 or 3.0. I guess it should not break anything, but who knows, someone might rely on there not being any stats...

**Comment by fabiospampinato** [comment]
I think that if somebody does than it's their mistake since this is not the expected behaviour of the plugin, nor is documented.

**Comment by backflip** [comment]
Sounds reasonable.

**Comment by backflip** [comment]
Published as 2.2.0
